annotation-utils/src/main/java/eu/europeana/annotation/config/SubtitleFormat.java

```diff
@@ -0,0 +1,29 @@
+package eu.europeana.annotation.config;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+@JacksonXmlRootElement(localName = "format")
+public class SubtitleFormat {
```

Comment:
this is a data model class, please follow the conventiosn and move it to .model package

Reply:
Done.